samples/caf_sensor_manager/src/modules/sensor_stub_gen.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <drivers/sensor_stub.h>
+
+#define MODULE sensor_stub_gen
+#include <caf/events/module_state_event.h>
+
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_SENSOR_STUB_GEN_LOG_LEVEL);
```

Comment:
Looking into logged messages im wondering if we need this at all? What do you think?

Reply:
It is used for the correctness test.

Reply:
Understood